plugins/vpc-bridge/config/netconfig.go

```diff
@@ -59,6 +63,21 @@ type netConfigJSON struct {
 	ServiceCIDR      string   `json:"serviceCIDR"`
 }
 
+// RuntimeConfig are the runtime options which will be populated dynamically by the runtime
+// based on the requested capability.
+// https://www.cni.dev/docs/conventions/#dynamic-plugin-specific-fields-capabilities--runtime-configuration
+type RuntimeConfig struct {
+	PortMappings []PortMapping `json:"portMappings,omitempty"`
+}
+
+// PortMapping contains the container port to host port mapping information.
+type PortMapping struct {
```

Comment:
It might be better to define this PortMapping struct and relevant functions in network/vpc/port.go. In addition to reusability, we also wouldn't have to import config package from vpc-bridge/network package.